demos/kitchen_sink/buildozer.spec

```diff
@@ -200,7 +200,7 @@ log_level = 2
 warn_on_root = 0
 
 # (str) Path to build artifact storage, absolute or relative to spec file
-# build_dir = ./.buildozer
+# build_dir = ~/build
 
 # (str) Path to build output (i.e. .apk, .ipa) storage
 # bin_dir = ./bin
```

Comment:
Why is this file edited? I would rather not mix concerns of pull requests, let's keep it simple

Reply:
Looks like when I added it I was building many apps at the same time and i want to save time to build app.